code/__HELPERS/logging.dm

```diff
@@ -82,6 +82,10 @@
 	if (config.log_pda)
 		diary << "\[[time_stamp()]]CHAT: [text]"
 
+/proc/log_comment(text)
+	if (config.log_pda) //reusing as not worth its own config option really

```

Comment:
No spacing between control statements and their brackets please. I do see that the above is also incorrect, and it is wrong.